backend/services/deployments/model/image.go

```diff
@@ -269,8 +269,12 @@ type MultipartGenerateImageMsg struct {
 
 func (msg MultipartGenerateImageMsg) Validate() error {
 	if err := validation.ValidateStruct(&msg,
-		validation.Field(&msg.Name, validation.Required),
-		validation.Field(&msg.DeviceTypesCompatible, validation.Required),
+		validation.Field(&msg.Name, validation.Required, lengthIn1To256),
+		validation.Field(
+			&msg.DeviceTypesCompatible,
+			validation.Required,
```

Comment:
Should we also limit the number of device types compatible?
```suggestion
			validation.Required,
			lengthIn1To256,
```

Reply:
Sure! Fixed in: 6f612d4c2d8ecb9e3d4f1246cd0755d77cdda11f